src/poliastro/plotting.py

```diff
@@ -375,3 +378,12 @@ def savefig(self, filename, **layout_kwargs):
             image=ext[1:], image_filename=basename,
             show_link=False,  # Boilerplate
         )
+
+
+def plot_solar_system(epoch=None):
```

Comment:
Just one last thing: can you add a docstring here so the new function gets properly documented? :)

Reply:
Okay :)